src/routes/explore/explore.styles.ts

```diff
@@ -97,3 +97,40 @@ export const Error = styled.div<{ shouldDisplay: boolean }>`
   display: ${({ shouldDisplay }) => (shouldDisplay ? 'flex' : 'none')};
   justify-content: center;
 `;
+
+export const Tooltip = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  max-width: 397px;
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity 200ms ease-in-out 250ms;
+  width: fit-content;
+`;
+
+export const TooltipArrow = styled.div`
```

Comment:
Tooltip arrow according to the design should be rounded a little, border-radius of 0 0 0 6px.

Reply:
I will fix it in the next PR because I'm working on the tooltip for levels.